docs/_data/sidebar-core.yml

```diff
@@ -279,6 +279,9 @@ options:
       - title: Core Extensions
         url: /apidocs/arrow-core-data/
 
+      - title: Core Retrofit
+        url: /apidocs/arrow-core-retrofit/
+
       - title: Integrations Retrofit Adapter
         url: /apidocs/arrow-integrations-retrofit-adapter/
```

Comment:
Do we need to keep this one?

Reply:
Thanks for the point @aballano :raised_hands: Yes, that module is still alive with the rest of the content:

```
arrow-integrations-retrofit-adapter/src/main/
└── kotlin
    └── arrow
        └── integrations
            └── retrofit
                └── adapter
                    ├── CallKind2CallAdapter.kt
                    ├── CallKindAdapterFactory.kt
                    ├── CallK.kt
                    ├── ResponseCallback.kt
                    └── syntax.kt
```

 `arrow-core-retrofit` just have part of that module now:

```
arrow-core-retrofit/src/main/
└── kotlin
    └── arrow
        └── retrofit
            └── adapter
                └── either
                    ├── ArrowEitherCallAdapter.kt
                    ├── ArrowResponseECallAdapter.kt
                    ├── eitherAdapter.kt
                    ├── EitherCallAdapterFactory.kt
                    └── ResponseE.kt
```

Note: I didn't show test directories in the trees :wink: